snuba/cli/replacer.py

```diff
@@ -108,6 +105,7 @@ def replacer(
     replacer = StreamProcessor(
         KafkaConsumer(
             build_kafka_consumer_configuration(
+                storage_name,
```

Comment:
Provided details below. Please pass the storage key and not the storage name around.